typescript/sdk/src/hook/read.ts

```diff
@@ -80,8 +81,16 @@ export class EvmHookReader implements HookReader {
     this.provider = this.multiProvider.getProvider(chain);
   }
 
-  public static stringifyConfig(config: HookConfig, space?: number): string {
-    return JSON.stringify(config, ethersBigNumberSerializer, space);
+  public static stringifyConfig(
+    config: HookConfig,
+    space?: number,
+    format: 'json' | 'yaml' = 'yaml',
+  ): string {
+    const json = JSON.stringify(config, ethersBigNumberSerializer, space);
+    if (format === 'json') {
+      return json;
+    }
+    return yamlStringify(JSON.parse(json), null, space);
```

Comment:
Why are you running it through JSON first? yaml can take arbitrary data just like JSON.stringify can

Reply:
was a quick way to get the bignumber reducer working for yaml ouput - but haven't yet dug into exactly why the current reducer doesn't play nice

Reply:
afaict this isnt specific to `EvmHookReader` or `HookReader` at all
can we pull this out into a generic object util in the CLI? and keep yaml out of the SDK?

Reply:
> was a quick way to get the bignumber reducer working for yaml ouput 
That's a good reason, feel free to leave it as is but maybe add a comment